crates/ty_python_semantic/src/semantic_index/builder.rs

```diff
@@ -2373,6 +2373,19 @@ where
                 }
                 walk_expr(self, expr);
             }
+            ast::Expr::Call(ast::ExprCall { func, .. }) if !self.source_type.is_stub() => {
+                let expression = self.add_standalone_expression(func);
```

Comment:
I'm not adding the new constraints in stub files now.
Current intention is just to please CI, until the visibility constraints refactor lands.

However, I do think that, this work of evaluating the `ReturnsNever` constraints is unnecessary in stub files?